CMakeLists.txt

```diff
@@ -524,6 +524,16 @@ endif()
 # Do this before consistency checks for added third-party libraries
 include( "src/Experimental.cmake" )
 
+cmd_option( ${_OPT}use_ffmpeg
+   "Use ffmpeg library [loaded, off]"
+   "loaded"
+   STRINGS "loaded" "off"
+)
+
+if( ${_OPT}use_ffmpeg )
+   set(USE_FFMPEG Yes)
+endif()
+
```

Comment:
This change of CMakeLists.txt should be in the previous commit "Use lib-ffmpeg-support"